### PR TITLE
Random choose FCP devices

### PR DESCRIPTION
**Pull request by changzhi1990**
Change the way to allocate FCP devices. Use random way instead of
always return the first one. Because VM will be allocated the same FCP devices
if the FCP devices don't work.

### CODE REVIEW

File: zvmsdk/database.py [hunk 1]
```diff
@@ -491,7 +493,11 @@ def get_fcp_pair(self):
                 fcps = result.fetchall()
                 if not fcps:
                     break
-                fcp_list.append(fcps[0][0])
+                available_fcp_list.append(fcps)
+            random_seed = [0, len(fcps) - 1]
+            index = random.choice(random_seed)
```
Comment (bjhuangr) [inline zvmsdk/database.py:498]: choice the first one or last one? Is it your design?

File: zvmsdk/database.py [hunk 2]
```diff
@@ -491,7 +493,11 @@ def get_fcp_pair(self):
                 fcps = result.fetchall()
                 if not fcps:
                     break
-                fcp_list.append(fcps[0][0])
+                available_fcp_list.append(fcps)
+            random_seed = [0, len(fcps) - 1]
```
Comment (dongyanyang) [inline zvmsdk/database.py:497]: you cann't assume the two paths has the same number of available FCPs

**Comment by changzhi1990** [comment]
Ran 640 tests in 1.092s

OK (skipped=3)
py27 run-test: commands[1] | python -m unittest discover -v -s /root/changzhi/python-zvm-sdk/smtLayer/tests/unit
[17170] /root/changzhi/python-zvm-sdk$ /root/changzhi/python-zvm-sdk/.tox/py27/bin/python -m unittest discover -v -s smtLayer/tests/unit
test_getVM_directory_py3 (test_vmUtils.SMTvmUtilsTestCase) ... ok
test_getDiskPoolSpace3390 (test_getHost.SMTGetHostTestCase) ... ok
test_getDiskPoolSpace3390Unknown (test_getHost.SMTGetHostTestCase) ... ok
test_getDiskPoolSpace3390UnknownWithFlag (test_getHost.SMTGetHostTestCase) ... ok
test_getDiskPoolSpace9336 (test_getHost.SMTGetHostTestCase) ... ok
test_getDiskPoolSpace9336Unknown (test_getHost.SMTGetHostTestCase) ... ok
test_getDiskPoolSpace9336UnknownWithFlag (test_getHost.SMTGetHostTestCase) ... ok
test_getReservedMemSize (test_makeVM.SMTMakeVMTestCase) ... ok
test_getReservedMemSize_equal_size (test_makeVM.SMTMakeVMTestCase) ... ok
test_getReservedMemSize_gap_G (test_makeVM.SMTMakeVMTestCase) ... ok
test_getReservedMemSize_invalid_suffix (test_makeVM.SMTMakeVMTestCase) ... ok
test_getReservedMemSize_max_less_than_initial (test_makeVM.SMTMakeVMTestCase) ... ok

----------------------------------------------------------------------
Ran 12 tests in 0.003s

OK
_______________________________________________________ summary ________________________________________________________
  pep8: commands succeeded
  py27: commands succeeded
  congratulations :)

**Comment by jichenjc** [comment]
please cherry pick, thanks

**Pull request merged**